README.md

```diff
@@ -16,42 +64,72 @@ The [v4_beta_deprecated](https://github.com/aws/aws-iot-device-sdk-embedded-C/tr
+
+For example, a second release in June 2021 would be 202106.01. Although the SDK releases have moved to date-based versioning, each library within the SDK will still retain semantic versioning. In semantic versioning, the version number itself (X.Y.Z) indicates whether the release is a major, minor, or point release. You can use the semantic version of a library to assess the scope and impact of a new release on your application. 
+
+## Cloning
+This repo uses [Git Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) to bring in dependent components.
+
+Note: If you download the ZIP file provided by GitHub UI, you will not get the contents of the submodules (The ZIP file is also not a valid git repository).  If you download from the [202009.00 Release Page](https://github.com/aws/aws-iot-device-sdk-embedded-C/releases/tag/202009.00) page, you will get the entire repository (including the submodules) in the ZIP file, aws-iot-device-sdk-embedded-c-202009.00.zip.  
+
+
+To clone using HTTPS:
+```
+git clone https://github.com/aws/aws-iot-device-sdk-embedded-C.git --recurse-submodules
+```
+Using SSH:
+```
+git clone git@github.com:aws/aws-iot-device-sdk-embedded-C.git --recurse-submodules
+```
+
+If you have downloaded the repo without using the `--recurse-submodules` argument, you need to run:
+```
+git submodule update --init --recursive
+```
+
+
 ## Building and Running Demos
 
   
+The libraries in this SDK are not dependent on any operating system. However, the demos for the libraries in this SDK are built and tested on a Linux platform. This SDK builds with [CMake](https://cmake.org/), a cross-platform build tool.
 
-The libraries in this SDK are not dependent on any operating systems. However, the demos for the libraries in this SDK are built and tested on a Linux platform. This SDK builds with [CMake](https://cmake.org/), a cross-platform build tool.
 
+### Prerequisites
 
+- CMake 3.13.0 or later 
 
-### Prerequisites
+- C90 compiler
 
-- CMake 3.13.0 or later and a C90 compiler.
+- Although not a part of the C90 standard, `stdint.h` is required for fixed-width integer types (e.g int32_t).
 
 - A supported operating system. The ports provided with this repo are expected to work with all recent versions of the following operating systems, although we cannot guarantee the behavior on all systems.
     - Linux system with POSIX sockets and timer APIs. (CI tests on Ubuntu 18.04).
         - On Linux systems, installation of OpenSSL development libraries and header files, *version 1.1.0 or later*, are required. The OpenSSL development libraries are usually called something like `libssl-dev` or `openssl-devel` when installed through a package manager.
-        - Although not a part of the C90 standard, `stdint.h` is required for fixed-width integer types (e.g int32_t).
+        
 
 
 
 ### AWS IoT Account Setup
 
 It is required to setup an AWS account and access the AWS IoT Console for running demos and tests. Follow the links to:
```

Comment:
This is an awkward sentence.  How about "The demos and tests require an AWS account."

Reply:
The focus of the statement is on the reader for _needing_ to create an AWS account. I have updated the statement to use first-person voice as "You need to setup an AWS..."